Apps/W1/EDocumentConnectors/Continia/app/src/API Requests/ContiniaAPIURLMgt.Codeunit.al

```diff
@@ -0,0 +1,233 @@
+    var
+        TechnicalResponseUrlLbl: Label '%1/documents/%2/technical_response.xml', Locked = true;
+    begin
+        exit(StrSubstNo(TechnicalResponseUrlLbl, CdnBaseUrl(), GetGuidAsText(DocumentGuid)))
+    end;
+
+    internal procedure BusinessResponseUrl(DocumentGuid: Guid): Text
+    var
+        BusinessResponseUrlLbl: Label '%1/documents/%2/business_responses.xml', Locked = true;
+    begin
+        exit(StrSubstNo(BusinessResponseUrlLbl, CdnBaseUrl(), GetGuidAsText(DocumentGuid)))
+    end;
+
+    internal procedure PartnerAccessTokenUrl(): Text
+    var
+        PartnerAccessTokenUrlLbl: Label '%1/partner/PartnerZoneLogin', Locked = true;
+    begin
+        exit(StrSubstNo(PartnerAccessTokenUrlLbl, COBaseUrl()));
+    end;
+
+    internal procedure PartnerZoneUrl(): Text
+    var
+        PartnerAccessTokenUrlLbl: Label '%1/partner/PartnerZoneConnect', Locked = true;
+    begin
+        exit(StrSubstNo(PartnerAccessTokenUrlLbl, COBaseUrl()));
+    end;
+
+    internal procedure ClientAccessTokenUrl(): Text
+    var
+        ClientAccessTokenUrlLbl: Label '%1/oauth/token/', Locked = true;
+    begin
+        exit(StrSubstNo(ClientAccessTokenUrlLbl, COBaseUrl()));
+    end;
+
+    internal procedure ClientEnvironmentInitializeUrl(): Text
+    var
+        ClientEnvironmentInitializeUrlLbl: Label '%1/core/initializeV3', Locked = true;
+    begin
+        exit(StrSubstNo(ClientEnvironmentInitializeUrlLbl, COBaseUrl()));
+    end;
+
+    internal procedure UpdateSubscriptionUrl(): Text
+    var
+        UpdateSubscriptionUrlLbl: Label '%1/core/UpdateSubscription', Locked = true;
+    begin
+        exit(StrSubstNo(UpdateSubscriptionUrlLbl, COBaseUrl()));
+    end;
+
+    internal procedure GetSubscriptionUrl(): Text
+    var
+        GetSubscriptionUrlLbl: Label '%1/core/GetSubscriptions', Locked = true;
+    begin
+        exit(StrSubstNo(GetSubscriptionUrlLbl, COBaseUrl()));
+    end;
+
+    internal procedure GetAcceptCompanyLicenseUrl(): Text
+    var
+        GetAcceptCompanyLicenseUrlLbl: Label '%1/core/AcceptCompanyLicense', Locked = true;
+    begin
+        exit(StrSubstNo(GetAcceptCompanyLicenseUrlLbl, COBaseUrl()));
+    end;
+
+    internal procedure GetUpdateCompanyInfoUrl(): Text
+    var
+        GetUpdateCompanyInfoUrlLbl: Label '%1/core/UpdateInvoicingInformation', Locked = true;
+    begin
+        exit(StrSubstNo(GetUpdateCompanyInfoUrlLbl, COBaseUrl()));
+    end;
+
+    local procedure COBaseUrl() Url: Text
+    var
+        Handled: Boolean;
+    begin
+        OnGetCOBaseUrl(Url, Handled);
+        if Handled then
+            exit(Url);
+
+        exit('https://auth.continiaonline.com/api/v1');
+    end;
+
+
+    internal procedure CdnBaseUrl() Url: Text
+    var
+        EnvironmentInformation: Codeunit "Environment Information";
+        Handled: Boolean;
+        LocalizedBaseUrl: Text;
+    begin
+        OnGetCdnBaseUrl(Url, Handled);
+        if Handled then
+            exit(Url);
```

Comment:
We cant do subscribers for tests. 
Is there any alternative way?

Or use the new HTTPClientHandler in the tests
Much simpler to implement than MockService now that we have it.

![image](https://github.com/user-attachments/assets/dc0e7660-8b24-4492-b8eb-32d5d2d4da21)
![image](https://github.com/user-attachments/assets/e60473b8-d452-41ad-a59c-1f13812a890f)